File: src/main/java/training/katas/PyramidArray/PyramidArray.dart
List<List<int>> pyramid(int n) {
  if (n == 0) {
    return new List();
  }
  var result = new List.filled(n, [], growable: false);
  for (var i = 1; i <= n; i++) {
    result[i - 1] = List<int>.filled(i, 1, growable: false).cast<int>();
  }
  return result.cast<List<int>>();
}

File: src/main/java/training/katas/PyramidArray/pyramid_array.js
function pyramid(n) {
    let result = [];
    if (n == 0) {
        return result;
    }
    for (let i = 1; i <= n; i++) {
        result.push(Array.from({ length: i }, (v, j) => 1));
    }
    return result;
}
module.exports = { pyramid };